projects/web/src/pages/ApiReferencePage.vue

```diff
@@ -38,5 +38,6 @@ const configuration = reactive<ReferenceConfiguration>({
   </div> -->
   <ApiReference
     :configuration="configuration"
-    @changeTheme="(theme: ThemeId) => (configuration.theme = theme)" />
+    @changeTheme="(theme: ThemeId) => (configuration.theme = theme)">
```

Comment:
```suggestion
    :hasEditor="false"
    @changeTheme="(theme: ThemeId) => (configuration.theme = theme)">
```
Try passing `hasEditor`  = false here to see it with no editor.